apps/miiverse-api/README.md

```diff
@@ -0,0 +1,12 @@
+# ⚡ juxt-api
+
+This is the API server for Juxtaposition. It powers both the Juxt frontends and the in-game Miiverse integrations.
```

Comment:
This is inaccurate. This is not a "Juxt" API, since it's not specific to us. It's an implementation of the Miiverse/Olive API, which is a standard used by Nintendo Network games. It largely has nothing to do with Juxtaposition outside of the data source and this repos association with us, you can take this server and hook it up to any database that has the expected schema and it will work just fine regardless of what project is used for the applet frontends (which *is* Juxtaposition). That's why the original repo was named `miiverse-api`

Reply:
Updated the texts